migration/v0.36/apply/11-kube-prometheus-stack.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
should this script be in v0.37 apps version?

Reply:
I asked @Xartos and he agrees that this could be moved to v0.37! :+1: